### PR TITLE
Upgrade to 0.6.0~ynh1

### LINKED ISSUE #22: Implement LDAP authentication

### LINKED ISSUE #61: v0.6.0 is out
Hi,

Now Plume 0.6.0 is out.
https://git.joinplu.me/Plume/Plume/releases/tag/0.6.0

It includes LDAP feature as said at #22!

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Fix #61*
- *Fix #22*

## Solution
- *Upgrade to 0.6.0~ynh1*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/plume_ynh%20PR62%20(yalh76)/)

### CODE REVIEW

File: conf/.env [hunk 1]
```diff
@@ -45,3 +45,12 @@ MAIL_HELO_NAME=no-reply@__DOMAIN__
...
 #PLUME_LOGO_512=icons/trwnh/paragraphs/plumeParagraphs512.png
+
+## LDAP CONFIG ##
+# the object that will be bound is "${USER_NAME_ATTR}=${username},${BASE_DN}"
+LDAP_ADDR=ldap://127.0.0.1:389
+LDAP_BASE_DN="ou=users,dc=yunohost,dc=org"
+LDAP_USER_NAME_ATTR=uid
+LDAP_USER_MAIL_ATTR=mail
+LDAP_TLS=false
+
```
Comment (igalic) [inline conf/.env:56]: do you folks use ldap?

**Review by igalic — COMMENTED** [review]
🙋🏻‍♀️

File: manifest.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
         "en": "Federated blogging application",
         "fr": "Une application de blogging fédérée."
     },
-    "version": "0.5.0~ynh1",
+    "version": "0.6.0~ynh1",
     "url": "https://baptiste.gelez.xyz/",
```
Comment (igalic) [inline manifest.json]: this url should be https://joinplu.me/

**Review by igalic — CHANGES_REQUESTED** [review]

File: conf/.env (hunk 1 above)
Comment (yalh76) [reply conf/.env:56]: yes, we are using openldap

File: manifest.json (hunk 2 above)
Comment (yalh76) [reply manifest.json]: fixed ^^

**Comment by yalh76** [comment]
Having some trouble to build the arm64 version ... So waiting to build it before releasing

**Comment by yalh76** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Review by alexAubin — APPROVED** [review]
LGTM

**Comment by alexAubin** [comment]
Lets uuuuuuuh meeeerge then :eyes: ?

**Comment by yalh76** [comment]
> 
> 
> Lets uuuuuuuh meeeerge then 👀 ?

was waiting to be abble to build arm64 version....
But now it's ready ^^

**Pull request merged**